lua/lvim/lsp/peek.lua

```diff
@@ -65,7 +72,7 @@ local function preview_location_callback(result)
 
   local opts = {
     border = "rounded",
-    context = 10,
+    context = lvim.lsp.peek.max_height,
```

Comment:
why using a different property here? Why not just name it context rather than masking it?

Reply:
yeah i should do thata 👍